input/fsh/models.fsh

```diff
@@ -30,14 +30,13 @@ criteria are defined by FHIRPath expressions.
   Name of the view definition, must be in a database-friendly format.
 """
 * name obeys sql-name
-* version 0..1 string "Business version of the view definition"
 * title 0..1 string "Name for this view definition (human friendly)" """
   A optional human-readable description of the view.
 """
+* meta 0..1 Meta "Metadata about the view definition"
```

Comment:
Adding this so ViewDefinitions can specify their profiles the same way as resources do.